__tests__/app.test.js

```diff
@@ -44,3 +45,53 @@ describe('/api/topics', () => {
          })
         });
     })
+   
+    describe('GET /api/articles/:article_id', () => {
+      it('responds with the requested article 1', () => {
+          return request(app)
+          .get('/api/articles/1')
+          .expect(200)
+          .then((response)=>{
```

Comment:
While this test is fine, it could be a bit better: We know the specific article which is being returned here and therefore could have asserted that we get the expected values back too EG: expect(response.body.author).toBe(<whatever the author's name actually is>)